foursight_core/react/api/react_api_base.py

```diff
@@ -36,7 +36,7 @@ def create_response(http_status: int = 200,
                         body: Union[dict, list] = None,
                         headers: dict = None,
                         content_type: str = JSON_CONTENT_TYPE) -> Response:
-        if not body:
+        if body is None:
```

Comment:
It was fixing a case where I was returning an empty *list* but since it was not passing the ``if not body`` in that case it went on to use `{}` which the UI code was not expecting (expecting a list not an object). Will add comments.